File: swagger-client/dart/lib/model/error_message.dart
part of swagger.api;

class ErrorMessage {
  /* The response code associated with the error. */
  int code = null;
/* The details or description of the error. */
  String message = null;

  ErrorMessage();

  @override
  String toString() {
    return 'ErrorMessage[code=$code, message=$message, ]';
  }

  ErrorMessage.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    code = json['code'];
    message = json['message'];
  }

  Map<String, dynamic> toJson() {
    return {
      'code': code,
      'message': message
     };
  }

  static List<ErrorMessage> listFromJson(List<dynamic> json) {
    return json == null ? new List<ErrorMessage>() : json.map((value) => new ErrorMessage.fromJson(value)).toList();
  }

  static Map<String, ErrorMessage> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, ErrorMessage>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new ErrorMessage.fromJson(value));
    }
    return map;
  }
}

File: swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/ErrorMessage.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// The resposne when an error occurs.
  /// </summary>
  [DataContract]
  public class ErrorMessage {
    /// <summary>
    /// The response code associated with the error.
    /// </summary>
    /// <value>The response code associated with the error.</value>
    [DataMember(Name="code", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "code")]
    public int? Code { get; set; }

    /// <summary>
    /// The details or description of the error.
    /// </summary>
    /// <value>The details or description of the error.</value>
    [DataMember(Name="message", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "message")]
    public string Message { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ErrorMessage {\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Message: ").Append(Message).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
